handbook/customers.md

```diff
@@ -1,63 +1,63 @@
 
 ### Missed Zoom Meeting | Etiquette with Customers
 This is a tutorial on how to respond when someone from outside the company misses a call.
 
-#### 48 hours before a meeting
+#### 48 hours before a meeting 
 
 L1: Check the calendar invite to determine if the customer has accepted the invitation.
 
-L2: If the customer has not accepted the invitation please reach out through appropriate channels to confirm attendance (Ex: EAs, Email, Slack).
+L2: If the customer has not accepted the invitation, please reach out through appropriate channels to confirm attendance (ex: EAs, email, Slack). 
 
 L3: Test the Zoom Meeting link to ensure that it is working.
 
 L4: Ensure that agenda documents are attached and accessible to meeting attendees (as appropriate to the situation).
 
 #### Day of meeting
 
-L1: Join meeting 2-3 minutes prior to the meeting start time.
+L1: Join the meeting two to three minutes before the meeting start time.
 
-L2: If the customer does not join the call after 3-minutes, contact the customer with:
+L2: If the customer does not join the call after three minutes, contact the customer by:
 
 - Slack, if we have a shared channel.
-- Email, using the email address from the calendar invite.
+- email, using the email address from the calendar invite.
 - LinkedIn, send a direct message.
-- Phone, try finding their number to text and/or call (as appropriate to the device type: landline vs cell phone).
+- phone, try finding their number to text and/or call (as appropriate to the device type: landline vs. cell phone).
 
-L3: In these communications to the customer remember to approach the situation with empathy. Anything could have happened.
+L3: In these communications to the customer, remember to approach the situation with empathy. Anything could have happened.
 
-L4: Be ready to supply an alternative date and time to reschedule the call. Suggest 2-3 options for the customer to choose from.
+L4: Be ready to supply an alternative date and time to reschedule the call. Suggest two to three options from which the customer can choose.
 
 L5: Ensure that contact information is accurate and that meeting invites can be received and accessed by the customer.
 
-L6: Repeat back to the customer the newly agreed upon date and time, as well as the contact information.
+L6: Repeat back to the customer the newly agreed-upon date and time, and the contact information.
 
 L7: Congratulations, you’re ready to set up a new call.
 
-### Next steps after a customer conversation
-After a customer conversation, it can sometimes feel like there are 1001 things to do, but it can be hard to know where to start.  Here are some tips:
+### Next steps after a customer conversation 
+After a customer conversation, it can sometimes feel like there are 1,001 things to do, but it can be hard to know where to start.  Here are some tips: 
```

Comment:
@tgauda and @mike-j-thomas, Should all of this be moved under "Support process"? It says "Here are some tips:" and then a new section with a bigger heading starts. Is there data missing?

Reply:
@Desmi-Dizney, it looks like an incomplete section to me. If there's content missing, I suggest that we remove this paragraph for now. @tgauda, please can you confirm? Thanks.